### DISCUSSION TITLE
Update to Node 16 in all projects

### DISCUSSION THREAD

**Discussion by eddybrando** [Qiskit/qiskit.org#2269] [Ideas]
Now that Node v16 is LTS (thanks @abdonrd for the heads up), should we update all projects?

I guess we'd need to update the **package.json** `engine` attribute where applicable, regenerate the **package-lock.json** files and also update the CI/CD pipelines.

Would there be anything else?

**Comment by abdonrd** [comment]
Agree to update the projects! 🚀

**Reply by korgan00** [reply]
Agree to update

**Comment by eddybrando** [comment]
We'll be tracking the update in https://github.com/Qiskit/qiskit.org/issues/2275.
